server/meta.test.js

```diff
@@ -1399,3 +1399,67 @@ test("should error when the origin parameter is not just the origin", () => {
     throw new Error("Expected error to be thrown");
   }
 });
+
+describe("Utilities", () => {
```

Comment:
Rather than checking every utility every time, I figured we could just check that they're equal and go from there. But it might not hurt to add specific assertions for each utility to each test case.

I started there, but it ended up being so many cases that it seemed redundant.